### ISSUE TITLE
Doesn't work on Mac OS

### ISSUE THREAD

**Issue by kieranhogg** [kevlar1818/is_my_password_pwned#1]
It needs md5sha1sum installed to run, but once that is installed, there are no results despite the API result indicating there are:

Reminder: This tool does not check password strength!
-n Type a password to check:

Hash prefix: 8317d
Hash suffix: 6fd696af28c7d921ceeeae536c9bd572254

Looking up your password...
 FFC52A365E284FBD188865D8CE6B39C6029:1939
Your password appears in the Pwned Passwords database 0 time(s).
Your password isn't pwned, but that doesn't necessarily mean it's secure!

**Comment by kevin-hanselman** [comment]
Is OpenSSL installed on Mac OS?

i.e. `openssl sha1`?

**Comment by kevin-hanselman** [comment]
@kieranhogg Please test the latest commit and report back. Thanks for kicking the tires in Mac land!

**Comment by kieranhogg** [comment]
openssl does indeed work. Latest version gives:

```sh pwned_pass.sh
Reminder: This tool does not check password strength!
Type a password to check:
Hash prefix:
Hash suffix:

Looking up your password...
usage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]
Your password appears in the Pwned Passwords database 0 time(s).
Your password isn't pwned, but that doesn't necessarily mean it's secure!```

**Comment by 1ne** [comment]
Seems like a behaviour issue with OpenSSL on macOS. Changed https://github.com/kevlar1818/is_my_password_pwned/blob/750e64561389de7a03d0f1596a0b6ebe51172497/pwned_pass.sh#L32 to one of the following it started working.

hash=$(echo -n $password | sha1sum | cut -f1 -d" ")
or
hash=$(echo -n $password | sha1sum | awk '{print $1}')

**Comment by kevin-hanselman** [comment]
We'll try to get Mac OS working in #4

**Comment by kevin-hanselman** [comment]
Fixed in #4.
